### DISCUSSION TITLE
No sound being played.

### DISCUSSION THREAD

**Discussion by danyislost** [iptv-org/community#1058] [Q&A]
I just recently discovered IPTV and I was trying to watch CaracolTV in VLC, and there is video but no sound. With almost every other channel there is sound and video. Any fixes? 
Here is the link to the playlist: https://iptv-org.github.io/iptv/countries/co.m3u
The channel's name is "Caracol"

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-report-a-broken-stream
